test/e2e-tests.sh

```diff
@@ -32,7 +32,7 @@ source "$(dirname "$0")/e2e-common.sh"
 
 initialize $@ --skip-istio-addon
 
-echo "Running tests with Multi Tenant Channel Based Broker"
-go_test_e2e -timeout=30m -parallel=12 ./test/e2e -brokerclass=MTChannelBasedBroker -channels=messaging.knative.dev/v1beta1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel,messaging.knative.dev/v1:Channel,messaging.knative.dev/v1:InMemoryChannel -sources=sources.knative.dev/v1alpha2:ApiServerSource,sources.knative.dev/v1alpha2:ContainerSource,sources.knative.dev/v1alpha2:PingSource || fail_test
+echo "Running E2E tests with Multi Tenant Channel Based Broker"
```

Comment:
```suggestion
echo "Running E2E tests for: Multi Tenant Channel Based Broker, Channel (v1beta1, v1), InMemoryChannel (v1beta1, v1) , ApiServerSource (v1beta1, v1), ContainerSource (v1alpha2, v1) and PingSource (v1beta1)"
```

Reply:
@devguyio so, we hammer the same in there too? Not sure that it's a really helpful 

B/c I am sure, that's always forgotten when we update the real scrpt, not just the markup

Reply:
@devguyio it just makes it easier to see which ones we're running, but I understand your POV. It least we remove the `with Multi Tenant Channel Based Broker` cause it gives wrong impression it does only that. So either

- we report all (whether hard coded or dynamic by having the channels and sources list as a variable)
- we just print a generic message